lib/car.rb

```diff
@@ -0,0 +1,26 @@
+require 'pry'
+class Car
+  attr_reader :name,
+              :make,
+              :model,
```

Comment:
Since you are defining methods for make (line 15) and model (line 19), it doesn't make sense to also declare them with `attr_reader`. `attr_reader` is used when you when you want to create "getter" methods on attributes that will be accessed outside of the Car class but we don't want to manually write out the method. So for `monthly_payment` it works because ruby is creating the following method for us:

```
def monthly_payment
  @monthly_payment
end
```

So since you are defining `make` and `model` already you don't need them to be created by ruby.